package-res/ccc/core/base/axis/color-axis.js

```diff
@@ -47,10 +52,16 @@ def('pvc.visual.ColorAxis', pvc_Axis.extend({
             // and if so, transform the color scheme.
             // If the user specified the colors,
             // do not apply default color transforms...
+
+            // CDF603 
```

Comment:
Shouldn't color transform still be applied to any colors for new categories, those taken from the base scale?